File: Program-Design/Python/Python-100/n53.py
n=int(input())
x=[]
for i in range(0,n):
    x.append(int(input()))
def square(x1):
    for i in range(0,len(x1)):
        x1[i]=pow(x1[i],2)
    return x1
square(x)
for i in range(0,len(x)):
    print(x[i],end="\t")
print()

File: Program-Design/Java/Java-100/n53.java

import java.util.Scanner;

public class n53 {
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int n = scn.nextInt();
        int[] a = new int[n];
        for (int i = 0; i < n; i++) {
            a[i] = scn.nextInt();
            System.out.print(square(a[i]) + "\t");
        }
        System.out.println();
    }
    public static int square(int ma){
        ma= (int) Math.pow(ma,2);
        return ma;
    }
}
